### DISCUSSION TITLE
Error fetching MediaItem using gatsby-source-wordpress

### DISCUSSION THREAD

**Discussion by allankiezel** [gatsbyjs/gatsby#35331] [Help]
Add new posts with featured images is now throwing an error fetching MediaItems. If i set the featured image to an existing image it works fine. It seems to happen now on new images. The error doesn't give much info, so I'm not sure if it's the amount or something else. I've tried setting GATSBY_CONCURRENT_DOWNLOAD to different values with no change in the result. All other images download except this one and I tried uploading a different image and still same issue.

info pushing Media Item query #0, digest: b3b2babf01254ce3418a028ad5c0104c to the end of the request queue.
info Previously retried 2 times already.
info pushing Media Item query #0, digest: b3b2babf01254ce3418a028ad5c0104c to the end of the request queue.
info Previously retried 3 times already.
info pushing Media Item query #0, digest: b3b2babf01254ce3418a028ad5c0104c to the end of the request queue.
info Previously retried 4 times already.
info

already re-queued Media Item query #0, digest: b3b2babf01254ce3418a028ad5c0104c 5 times :( sorry.
Try lowering process.env.GATSBY_CONCURRENT_DOWNLOAD.
It's currently set to undefined

 ERROR 

Request failed with status code 502

  Error: Request failed with status code 502
  
  - createError.js:16 createError
    [gatsby]/[axios]/lib/core/createError.js:16:15
  
  - settle.js:17 settle
    [gatsby]/[axios]/lib/core/settle.js:17:12
  
  - http.js:269 IncomingMessage.handleStreamEnd
    [gatsby]/[axios]/lib/adapters/http.js:269:11
  
  - task_queues.js:80 processTicksAndRejections
    internal/process/task_queues.js:80:21

**Comment by jsrhodes15** [comment]
@allankiezel Did you ever find a solution to this? I have been banging my head against a wall for almost a week now. I have the EXACT same problem and no idea what is going on.

**Comment by justlevine** [comment]
[Getting reports](https://github.com/AxeWP/wp-graphql-rank-math/issues/15) about this too, although I dont have DB access to diagnose the issue.

@allankiezel @jsrhodes15 only thing I can guess is a MediaItem was removed but still being set as a featured image, or (but less likely) the item exists in the Media Library database, but the file itself is missing. Did you check to see if you have any stray references?
